### DISCUSSION TITLE
Error in compilation while plotting using python script

### DISCUSSION THREAD

**Discussion by bha112** [idaholab/moose#22540] [Q&A Modules: General] [answered]
Hello,

Using this command in my desktop for compilation of the .json file
```
 **python ../../../python/make_histogram.py main_out.json* -v results:T_avg:value --xlabel 'Average Temperature'** 

Traceback (most recent call last):
  File "/home/lab_3/projects/moose/modules/stochastic_tools/examples/parameter_study/gold/../../../python/make_histogram.py", line 17, in <module>  import mooseutils .

ModuleNotFoundError: No module named 'mooseutils'
```
Its seems python script is not working (make_histogram.py). 

Could you please help me to solve the issue?

Bhavesh

**Comment by GiudGiud** [comment, marked as answer]
Hello

You need to add moose/python to your pythonpath so python knows where to find mooseutils

like this:
`
export PYTHONPATH=$PYTHONPATH:/home/lab_3/projects/moose/python
`

Guillaume

**Reply by bha112** [reply]
Thanks for your response.
